### ISSUE TITLE
Quick question about pre-training with one sentence

### ISSUE THREAD

**Issue by cloudygoose** [microsoft/MASS#20]
I have a quick question for your MASS paper, thanks for the good work and sharing your codes.

My question is that I want to confirm in your pre-training, your masked sentence is always ONE sentence, instead of several. That's my understanding from the paper.

I'm asking this because when thinking about some down-stream tasks such as summarization or dialogue, the input is usually several sentences, so it could be interesting to try pre-training with multiple sentences as input.

**Comment by tobyoup** [comment]
Thanks for your suggestion. Yes, it's one sentence in the paper. Currently we are conducting experiments with multiple masked sentences. We will update more if there is any progress.

**Comment by cloudygoose** [comment]
Got it, thanks!
